scripts/json_plugins.ts

```diff
@@ -79,7 +80,7 @@ for (let language in languages) {
       pluginSet.add(id);
     }
     const pluginPath = path.join(langPath, plugin);
-    execSync(`jsmin -l 3 --overwrite ${pluginPath}`);
+    execSync(`terser ${pluginPath} --config-file ${terserCfg} -o ${pluginPath}`);
```

Comment:
I think we should compress file first (move this line to above of the `require` line)

Reply:
I'll redo this via the `terser` api later.